python/nano/test/ray/test_ray_trainer.py

```diff
@@ -21,8 +21,8 @@
 import pytest
 import torch
 from pytorch_lightning import LightningModule
-from _train_torch_lightning import create_data_loader, data_transform
-from _train_torch_lightning import train_with_linear_top_layer
+from python.nano.test.pytorch.utils._train_torch_lightning import create_data_loader, data_transform
```

Comment:
I think the import should be `test.pytorch.utils` as the directory of `python` and `nano` are not python packages.

The reason why `python.nano.test.` works is because the `python -m pytest` will add current directory in `sys.path`